src/configuration.c

```diff
@@ -317,18 +317,20 @@ static void set_default_plugin(struct mptcpd_config *config,
  * @param[in]     plugins Comma separated list of plugins to load.
  */
 static void set_plugins_to_load(struct mptcpd_config *config,
-                                char *plugins)
+                                char const *plugins)
```

Comment:
Since the argument `plugins` is modified, please remove the `const` qualifier, and drop the cast that removes the `const` below.  It's better to have the code be `const`-correct in this case.
```suggestion
                                char *plugins)
```